src/Message.chpl

```diff
@@ -50,6 +312,7 @@ module Message {
         var nreader = newmem.reader();
         try {
             nreader.readf("%jt", msg);
+            writeln("%jt".format(msg));
```

Comment:
i think this `writeln` is leftover

Reply:
Yeah, added for a test and forgot to remove before pushing changes. Removed